packages/core/src/utilities/imageToWorldCoords.ts

```diff
@@ -23,12 +23,15 @@ export default function imageToWorldCoords(
 
   const {
     columnCosines,
-    columnPixelSpacing,
     rowCosines,
-    rowPixelSpacing,
     imagePositionPatient: origin,
   } = imagePlaneModule;
 
+  let { columnPixelSpacing, rowPixelSpacing } = imagePlaneModule;
+  // Use ||= to convert null and 0 as well as undefined to 1
+  columnPixelSpacing ||= 1;
+  rowPixelSpacing ||= 1;
+
```

Comment:
it would be good to have a console warn here too

Reply:
Not sure why you want that - it is a perfectly valid case, just one that we weren't handling correctly previously.